test/integration/integration_test.go

```diff
@@ -141,33 +142,43 @@ func FeatureContext(s *godog.Suite) {
 	s.Step(`^services "([^"]*)" rollout successfully$`,
 		minishift.rolloutServicesSuccessfully)
 
+	// steps for proxying
+	s.Step(`^user starts proxy server and sets MINISHIFT_HTTP_PROXY variable$`,
+		testProxy.SetProxy)
+	s.Step(`^user stops proxy server and unsets MINISHIFT_HTTP_PROXY variable$`,
+		testProxy.UnsetProxy)
+	s.Step(`^proxy log should contain "([^"]*)"$`,
+		proxyLogShouldContain)
+	s.Step(`^proxy log should contain$`,
+		proxyLogShouldContainContent)
+
 	// steps to verify `stdout`, `stderr` and `exitcode` of commands executed
-	s.Step(`([^"]*) should contain "([^"]*)"$`,
+	s.Step(`^(stdout|stderr|exitcode) should contain "([^"]*)"$`,
```

Comment:
:+1 I like that this is getting more explicit. This becomes more and more like Aruba. I like.